### ISSUE TITLE
num_features

### ISSUE THREAD

**Issue by SuperAttitude** [cyh1112/GraphNormalization#3]
Hello, thank you very much for your work. I noticed num_features in your code. I want to know what it means and how it is set in use.Thank you for your reply.

**Comment by cyh1112** [comment]
Hi, thanks for attention, 'num_features' means the number of node/edge feature channel in this work. and you can see more about 'num_features' in BatchNorm, InstanceNorm:

[https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm1d.html](https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm1d.html)
[https://pytorch.org/docs/stable/generated/torch.nn.InstanceNorm1d.html#torch.nn.InstanceNorm1d](https://pytorch.org/docs/stable/generated/torch.nn.InstanceNorm1d.html#torch.nn.InstanceNorm1d)

**Comment by cyh1112** [comment]
You can modify it in config file, such as ['configs/SBMs_node_clustering_GCN_CLUSTER_100k.json'](https://github.com/cyh1112/GraphNormalization/blob/master/configs/SBMs_node_clustering_GCN_CLUSTER_100k.json) line 27: net_params.hidden_dim

**Comment by SuperAttitude** [comment]
Thank you for your reply. I'll read it
